packages/components/src/entities/SampleAliquotsGridPanel.tsx

```diff
@@ -163,38 +76,43 @@ export const SampleAliquotsGridPanelImpl: FC<Props & InjectedQueryModels> = memo
         [createNotification]
     );
 
-    const exportOption = {
-        [EXPORT_TYPES.LABEL]: onPrintLabel,
-    };
+    const containerFilter = useMemo(() => getContainerFilterForLookups(), []);
 
-    const showPrintOption = showLabelOption && canPrintLabels;
+    const excludedMenuKeys = useMemo(() => {
+        const exclude = [SamplesEditButtonSections.IMPORT, SamplesEditButtonSections.EDIT_PARENT];
```

Comment:
The IMPORT exclusion here is for the "Add > Import from File" menu option right? I am not seeing that option here, but I am still seeing the "Edit > Update from File" option. Should excluding IMPORT hide them both?

Reply:
I'll defer the decision of "Update from File" based on IMPORT to a later time. There are currently some Biologics registry whose "Update from File" is not support but could be. We can handle the change when there is request to support those scenarios.